src/olympia/devhub/tests/test_views.py

```diff
@@ -1414,8 +1415,9 @@ def test_system_addon_not_allowed_not_mozilla(self, mock_validator):
     @mock.patch('olympia.devhub.tasks.run_addons_linter')
     @mock.patch('olympia.files.utils.get_signer_organizational_unit_name')
     def test_mozilla_signed_allowed(self, mock_get_signature, mock_validator):
-        user_factory(email='redpanda@mozilla.com')
+        user = user_factory(email='redpanda@mozilla.com')
```

Comment:
As above, would it make sense to change the domain of the email address?

Reply:
8c8f9a8a9f6980c7dc702160d08d47e2c32ee42e